### ISSUE TITLE
Slider Component does not announce "disabled"

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30939]
## Description
When using a screen reader this component does not announce that it is disabled.

This issue is covered in-depth by parent issue #30840, please check there for more details.

## React Native version:
0.63

**Comment by Simek** [comment]
@amarlette Slider component has been deprecated in the `0.59` release (https://github.com/react-native-community/releases/blob/master/CHANGELOG.md#deprecated-4) and have been extracted from the core. 

Please report the issue in this repository:
* https://github.com/callstack/react-native-slider

**Comment by fabriziobertoglio1987** [comment]
maybe this issues should be closed.

pr https://github.com/callstack/react-native-slider/pull/335 was merged and issue https://github.com/callstack/react-native-slider/issues/265 was closed

**Comment by fabriziobertoglio1987** [comment]
@blavalla 
I published https://github.com/callstack/react-native-slider/pull/354 to fix this issue. 
The PR is in DRAFT. I will complete the PR in a couple of days. 
Thanks :pray:
